src/App.tsx

```diff
@@ -181,6 +188,10 @@ function App() {
     }
   });
 
+  useEffect(() => {
+    setItemsNeeded(countItemsNeeded());
+  }, [omittedItems]);
```

Comment:
```suggestion
```

Reply:
just add `omittedItems` to the array on line 178.

Reply:
done & done